### DISCUSSION TITLE
How to disable :emoji: replace?

### DISCUSSION THREAD

**Discussion by antonmedv** [facebook/docusaurus#5450] [Q&A]
So code like `:key:` will be replaced with an emoji. Is there a way to disable it?

**Comment by slorber** [comment]
Hmmm, didn't even know we had that plugin by default: https://github.com/rhysd/remark-emoji

We should allow to disable it, but currently it's hardcoded, and removing it could be a breaking change for some users.

Tracked in https://github.com/facebook/docusaurus/issues/4625

What's your use-case for disabling this plugin?

**Reply by antonmedv** [reply]
I have a doc with a title: `dep:key:generate`. And key is replaced with 🔑 
🙃

**Reply by slorber** [reply]
🤪 

As a workaround you can use a code block or JSX:

```jsx
Hey `dep:key:generate` Hello

<span>{"Hey dep:key:generate Hello"}</span>
```

**Reply by lex111** [reply]
Besides above, you can also use zero-width space, eg `dep:&#8203;key:generate`

**Reply by slorber** [reply]
TIL about zero-width spaces 🤪

**Comment by antonmedv** [comment]
Is it still not possible? =)

**Reply by slorber** [reply]
still not possible for now

the only option is to apply a local patch to Docusaurus, for example with your package manager or patch-package

Line: https://github.com/facebook/docusaurus/blob/main/packages/docusaurus-mdx-loader/src/processor.ts#L97

**Reply by antonmedv** [reply]
I ended up with zero white space solution and simple script to replace those emojis.
